src/transaction.cpp

```diff
@@ -275,10 +281,11 @@ namespace mamba
         queue_free(&pkgs);
     }
 
-    auto MTransaction::to_conda() -> to_conda_type 
+    auto MTransaction::to_conda() -> to_conda_type
     {
-        to_install_type to_install_structured; 
+        to_install_type to_install_structured;
         to_remove_type to_remove_structured;
+        Console::instance().json_down("FETCH");
```

Comment:
I don't think this is the correct place. `to_conda` just turns the transaction to a JSON format that conda & mamba use to interface with each other.

Maybe we should have a dedicated function to print the transaction as JSON?